classes/evasys_synchronizer.php

```diff
@@ -302,14 +302,22 @@ public function notify_evaluation_responsible_person($dates, $newparticipantsadd
             "Falls Sie für diesen Kurs bereits eine E-Mail erhalten haben, wurden gerade neue Teilnehmer*innen ".
             "hinzugefügt oder der Zeitraum angepasst. Dies ist ggf. unten angegeben.\r\n\r\n";
 
+        $standardtime = false;
+        if ($dates != "Standard") {
+            $notifsubject .= ' SONDERWUNSCH';
```

Comment:
Sonderwunsch should be printed in the beginning of the subject. (The course fullname can be pretty long)